v2/proto/messages_go_1.7_test.go

```diff
@@ -10,10 +10,9 @@ import (
 	"time"
 )
 
-func TestProduceRequest(t *testing.T) {
+func TestProduceRequest17(t *testing.T) {
```

Comment:
what does the number `17` represent here?

Reply:
golang version 1.7